.github/actions/bumpVersion/bumpVersion.js

```diff
@@ -1,15 +1,18 @@
 const {promisify} = require('util');
 const exec = promisify(require('child_process').exec);
-const fs = require('fs');
 const core = require('@actions/core');
 const github = require('@actions/github');
+const functions = require('../../libs/versionUpdater');
```

Comment:
Or better yet, since `versionUpdater` uses named exports, just destructure the named exports you use:

```
const {incrementVersion, ..., ...} = require('../../libs/versionUpdater');
```